src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -25,10 +25,10 @@ public void updateCart(@PathVariable Long cartId, @RequestBody ProductDto produc
     }
 
     @DeleteMapping(path = "/{id}")
-    public void deleteProduct(@PathVariable Long cartId, @PathVariable Long productId) {
+    public void deleteProduct(@PathVariable("id") Long cartId, @PathVariable Long productId) {
     }
 
     @PostMapping(path = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    public void createOrder(@PathVariable Long cartId) {
+    public void createOrder(@PathVariable("id") Long cartId) {
     }
-}
+}
```

Comment:
Brak implementacji metod

Reply:
Dlaczego w CartController masz w mapowaniu np. zamiast CartDto to ProductDto lub deleteProduct albo createOrder?

Reply:
> Dlaczego w CartController masz w mapowaniu np. zamiast CartDto to ProductDto lub deleteProduct albo createOrder?

Pewnie dlatego, że usuwamy produkt z koszyka i tworzymy na jego podstawie zamówienie. Natomiast jeśli chodzi o ProductDto to tutaj w rzeczywistej implementacji faktycznie powinno być CartDto getProducts()